calibration/calibrate_model.sh

```diff
@@ -349,8 +358,19 @@ if [[ -n $RANK ]]; then
     if [[ "$USE_EP" == "true" ]]; then
         EP_ARG="--use_expert_paral"
     fi
-    python3 step-5-unify_measurements.py -r $RANK -m $QUANT_DIR -o $QUANT_DIR $EP_ARG || (echo "Error in step 5" && exit 1)
+    python3 step-5-unify_measurements.py -r $RANK -m $QUANT_DIR -o inc_tmp/$MODEL_NAME/$DEVICE_TYPE/ $EP_ARG || (echo "Error in step 5" && exit 1)
```

Comment:
It will keep the original files before expanding, right?

Reply:
Yes, this change make sure the input/output folder is different to avoid potential read/write confilict.